src/globals.ts

```diff
@@ -365,7 +365,10 @@ export const DataManager = () => ({
             foldingBuffer: options.foldingBuffer,
         };
     },
-    addTimeReachedTrigger: (recordingLengthMicroSeconds: number) =>
+    addTimeReachedTrigger: (
+        recordingLengthMicroSeconds: number,
+        biasPercent = 10
```

Comment:
As far as I can see default value is not needed and not being used, if so consider
```suggestion
        biasPercent: number
```